src/Git2/Exception.cc

```diff
@@ -45,5 +46,4 @@ git2Throw(const int ret) {
   }
   return ret;
 }
-
```

Comment:
This line should be kept.

Reply:
Ok, updating it

Reply:
Also thank you so much for making this, it's been really helpful!

Reply:
Of course, I'm happy to hear that!